### DISCUSSION TITLE
The final output is interrupted because the C: drive has been fully occupied

### DISCUSSION THREAD

**Discussion by mazidong1** [NHERI-SimCenter/SimCenterCommon#231] [R2D] [answered]
Dear developers,

I wanted to simulate the response of over 100 buildings, but R2D covered the C drive space when it reached 50 buildings.

Can I change the path of the output result? However, I couldn't find the file to modify the path, please indicate, thanks!

**Comment by jinyan1214** [comment, marked as answer]
You should be able to modify it by changing the "Local Jobs Directory" in R2D preference.

**Reply by mazidong1** [reply]
Dear Jinyan,

Thank you very much for your answer, which enables me to continue with my upcoming research!

Best,

Zidong
